### ISSUE TITLE
can we create permanent table in spark for SHC?

### ISSUE THREAD

**Issue by lionelcao** [hortonworks-spark/shc#214]
Hi,
Do we have to build the dataframe first and then create the temp view/table?
I want to know is it possible to create permanent table in spark to use SHC & if Y how to specify the columns mapping between spark table and HBase table?

**Comment by weiqingy** [comment]
Yes. You need to build dataframe, and use `dataframe.read(or write).option(catalog) `to tell spark the columns mapping between spark table and HBase table (aka. catalog)

**Comment by lionelcao** [comment]
Em...Can we use statement like "create table ... using <data source> tableproperties ..." to create permanent table? I want to make all table ready in spark so that users can use them directly...

**Comment by weiqingy** [comment]
For "using xxx", it may hit some issues. (But, you can try it)

If you want to create permanent tables, you can save data from Hbase into a dataframe first, then use saveAsTable() creates a permanent, physical table.

**Comment by lionelcao** [comment]
Ya..I looked into saveAsTable function and looks like it will read data from HBase into DF and save the data as parquet format. Not only save the metadata of table just like the temp view...
